src/commands/Configuration/configure/automod/disable.ts

```diff
@@ -17,26 +18,26 @@ export default class extends SubCommand {
 		});
 	}
 
-	public async run(message: Message, args: string[]): Promise<void | Message> {
+	public async run(context: CommandContext, args: string[]): Promise<void | Message> {
 		const partArg = args[0];
 		if (!partArg) {
-			await databaseManager.enableAutomod(message.guild!.id, false);
-			await message.channel.send(await this.client.bulbutils.translate("automod_disabled", message.guild?.id, {}));
+			await databaseManager.enableAutomod(context.guild!.id, false);
+			await context.channel.send(await this.client.bulbutils.translate("automod_disabled", context.guild?.id, {}));
 		} else {
-			const partexec = /^(message|mention|website|invite|word|token)s?$|^word_?(token)s?$/.exec(partArg.toLowerCase());
+			const partexec = /^(context|mention|website|invite|word|token)s?$|^word_?(token)s?$/.exec(partArg.toLowerCase());
 			if (!partexec)
-				return message.channel.send(
-					await this.client.bulbutils.translate("event_message_args_unexpected", message.guild!.id, {
+				return context.channel.send(
+					await this.client.bulbutils.translate("event_message_args_unexpected", context.guild!.id, {
 						argument: partArg,
 						arg_expected: "part:string",
-						usage: "`website`, `invites`, `words`, `word_tokens`, `mentions` or `messages`",
+						usage: "`website`, `invites`, `words`, `word_tokens`, `mentions` or `contexts`",
```

Comment:
Contexts should be replaced with `messages` as there is no `contexts` automod category